R/sample.R

```diff
@@ -189,6 +189,26 @@ initial_parameters <- function(initial, model, rng, call = NULL) {
     }
     initial <- matrix(initial, n_chains, n_pars, byrow = TRUE)
   }
+
+  err <- initial < model$domain[, 1] | initial > model$domain[, 2]
```

Comment:
This is getting triggered only if initial was provided, since direct_sample_within.. deals with the case where it can't generate in-domain initial itself..?

Reply:
yeah, exactly